### ISSUE TITLE
PHP 7.3 - preg_replace(): Compilation failed: invalid range in character class at offset 24

### ISSUE THREAD

**Issue by ejunker** [atymic/twitter#250]
When using PHP 7.3 I get the error:
`preg_replace(): Compilation failed: invalid range in character class at offset 24`

It seems to be an issue in `Twitter.php` with the hashtag regex. 

```php
$patterns['hashtag'] = '(?:(?<=\s)|^)#(\w*[\p{L}-\d\p{Cyrillic}\d]+\w*)';
...
// Hashtag
$text = preg_replace('/'.$patterns['hashtag'].'/ui', '<a href="https://twitter.com/search?q=%23\\1" target="_blank">#\\1</a>', $text);
```

You can see example output for different versions of PHP at: https://3v4l.org/H56I0

**Comment by kawax** [comment]
```
$patterns['hashtag'] = '(?:(?<=\s)|^)#(\w*[\p{L}\-\d\p{Cyrillic}\d]+\w*)';
```

{L}- => {L}\\-

Add one backslash.

**Comment by atymic** [comment]
Hey :)

We've merged a PR to fix this in our fork
https://github.com/atymic/laravel-twitter

More Info:

I've created a fork (with issues enabled) & packagist release of the current version in the same namespace (unlike some of the other forks).

I plan to maintain this fork as i'm using it in a few projects.

https://github.com/atymic/laravel-twitter

I'd love to work with others on the package as well, so feel free to reach out.

If the original author does return, I'd be happy to merge our changes back into the mainline. If not, I plan to release v3 in a different namespace and fix a lot of the pre existing issues.
